### PR TITLE
feat: mirror nsolid

### PR DESCRIPTION
**Pull request by fengmk2**
closes https://github.com/cnpm/mirrors/issues/110

---
This change is [Reviewable](https://reviewable.io/reviews/cnpm/mirrors/111)

### CODE REVIEW

**Comment by fengmk2** [comment]
https://cnpmjs.org/mirrors/nsolid/latest/ 正在同步 ing

**Comment by fengmk2** [comment]
同步完成。

File: sync/nsolid.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const util = require('util');
+const NodeSyncer = require('./node');
+
+module.exports = NSolidSyncer;
```
Comment (popomore) [inline sync/nsolid.js]: 我也改成 S 大写

File: sync/nsolid.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+function NSolidSyncer(options) {
```
Comment (popomore) [inline sync/nsolid.js]: 为啥这里要继承一下，啥也没改。

**Comment by popomore** [comment]
+1

File: sync/nsolid.js (hunk 2 above)
Comment (fengmk2) [inline sync/nsolid.js]: 直接 copy alinode 的 syncer。。。

**Pull request merged**